### ISSUE TITLE
Script to star repos?

### ISSUE THREAD

**Issue by ghost** [dwyl/stars#17]
Is it possible to make a script to star all dwyl repos?

**Comment by iteles** [comment]
@markwilliamfirth It's great to assume that people will love us enough to want to do this, but I'm curious to see if there's anyone who can validate this issue 😆

**Comment by ghost** [comment]
I starred them all today 😄 😉 
⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ ⭐️ 

...also curious...

**Comment by nelsonic** [comment]
@markwilliamfirth yeah, it's easy enough to write the script you are describing. requires OAuth tho. which we have: https://github.com/dwyl/hapi-auth-github ... 😉
